cli.js

```diff
@@ -101,15 +101,59 @@ function loadLanguages() {
 	return new Set(languages);
 }
 
-function findEmail() {
+function findGitConfigEmail() {
 	let email;
 	try {
 		email = execa.sync('git', ['config', 'user.email']).stdout.trim();
-	} catch (_) {}
+	} catch {}
 
 	return email;
 }
 
+async function findEmail(existingSrc) {
```

Comment:
Refactors out email resolution to be consistent between the _new_ and _upgrade_ codepaths.  There are a few scenarios - e.g. upgrading an existing Code of Conduct that doesn't contain an email - that will result in the `undefined` scenario from #18.  This should force the prompt/error in all paths.